### PR TITLE
Add Karafun 2.2.0.86

### PR DESCRIPTION
**Pull request by MoshiBin**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/karafun.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "karafun" do
+  version "2.2.0,86"
+  sha256 "ca26e741588827b17b8ae4d72d3f713ba8aa029cbddea08eb8403a9bc8151b42"
+
+  url "https://c20.recis.io/sl/75651288c7/63683b44547cfdbafd795b67c7101da5/KaraFun_#{version.csv.first}.#{version.csv.second}.dmg",
```
Comment (bevanjkay) [inline Casks/karafun.rb:5]: I am not sure about the stability of this URL, particularly whether the additional values change between versions.

Comment (razvanazamfirei) [reply Casks/karafun.rb:5]: Unversioned url here: `https://www.karafun.com/download/mac.html`

Comment (bevanjkay) [reply Casks/karafun.rb:5]: @MoshiBin originally had the unversioned url here. I found the versioned URL and would prefer to use it if it remains stable. I suggest we try it first and see if/what breaks when a new version is pushed.

Comment (razvanazamfirei) [reply Casks/karafun.rb:5]: My bad, did not check the original commit! :) Sounds good!

**Comment by miccal** [comment]
Thank you @MoshiBin.

**Review by miccal — APPROVED** [review]

**Pull request merged**